data/mods/stadium/moves.ts

```diff
@@ -21,6 +21,12 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 			}
 		},
 	},
+	hyperbeam: {
+		inherit: true,
+		onMoveFail(target, source, move) {
+			source.addVolatile('mustrecharge', target, move, 'trapper');
```

Comment:
Why 'trapper'? Was this copy/pasted from a trapping move? Please don't tell me RBY is also like this...

Reply:
(`trapper` is a flag for modern trapping moves, which means "this effect goes away if the opponent switches out", which obviously doesn't apply to Stadium Hyper Beam.)

Reply:
It looks like no, it wasn't copy/pasted from RBY. Whew, I was afraid RBY needed to be fixed. Where did you get this from?

```suggestion
			source.addVolatile('mustrecharge');
```

Reply:
I don't remember where I copied it from, but somehow I didn't pay attention to the unnecessary extra arguments.